lms/urls.py

```diff
@@ -101,6 +101,7 @@
 
     url(r'^api/commerce/', include('commerce.api.urls', namespace='commerce_api')),
     url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls', app_name="credit", namespace='credit')),
+    url(r'^rss_proxy/', include('rss_proxy.urls', namespace='rss_proxy')),
```

Comment:
More along the lines of code philosophy as mentioned above, I guess -- just my opinion that we should be controlling all aspects of a feature via configuration flag, unless the feature is designed to be always-on for all possible Open edX operators, which seems like the minority case to me.